draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1539,23 +1562,27 @@ AEGIS-256 offers 256-bit message security against plaintext and state recovery,
+Protocols mandating a fully committing scheme without that restriction can provide the associated data as input to a cryptographic hash function and use the output as the `ad` parameter of the `Encrypt` and `Decrypt` functions. The selected hash function must ensure a minimum of 128-bit preimage resistance. An instance of such a function is SHA-256 {{!RFC6234}}.
 
-Under the assumption that the secret key is unknown to the attacker both AEGIS-128L and AEGIS-256 target 128-bit security against forgery attacks regardless of the tag size.
+Under the assumption that the secret key is unknown to the attacker, all AEGIS variants target 128-bit security against forgery attacks regardless of the tag size.
 
-Both algorithms MUST be used in a nonce-respecting setting: for a given `key`, a `nonce` MUST only be used once. Failure to do so would immediately reveal the bitwise difference between two messages.
+AEGIS algorithms MUST be used in a nonce-respecting setting: for a given `key`, a `nonce` MUST only be used once. Failure to do so would immediately reveal the bitwise difference between two messages.
 
-If tag verification fails, the decrypted message and wrong message authentication tag MUST NOT be given as output. As shown in {{VV18}}, even a partial leak of the plaintext without verification would facilitate chosen ciphertext attacks.
+If tag verification fails, the unverified plaintext and the computed message authentication tag MUST NOT be released. As shown in {{VV18}}, even a partial leak of the plaintext without verification would facilitate chosen ciphertext attacks.
 
 Every key MUST be randomly chosen from a uniform distribution.
 
 The nonce MAY be public or predictable. It can be a counter, the output of a permutation, or a generator with a long period.
 
-With AEGIS-128L, random nonces can safely encrypt up to 2<sup>48</sup> messages using the same key with negligible (~ 2<sup>-33</sup>, to align with NIST guidelines) collision probability.
+With AEGIS-128L and AEGIS-128X, random nonces can safely encrypt up to 2<sup>48</sup> messages using the same key with negligible (~ 2<sup>-33</sup>, to align with NIST guidelines) collision probability.
+
+With AEGIS-256 and AEGIS-256X, random nonces can be used with no practical limits.
 
-With AEGIS-256, random nonces can be used with no practical limits.
+AEGIS nonces match the size of the key. AEGIS-128L and AEGIS-128X feature 128-bit nonces, offering an extra 32 bits compared to the commonly used AEADs in IETF protocols. The AEGIS-256 and AEGIS-256X variants provide an even ampler space for nonces, surpassing the 192-bit requirement for secure utilization of random nonces without practical limitations.
```

Comment:
I would change `ampler`. Maybe something like `provide even larger nonces`.

Reply:
Having said that, I'm not sure if this belongs in this section. This sounds more like something for the introduction and somewhat repeats what's said directly above (no practical limits).

Reply:
`ampler` changed, thanks!

Maybe something like `even with nonces made of 192 random bits, 64 additional bits remain available to optionally encode additional information`?

Reply:
`an extra 32 bits` -> `32 extra bits` maybe?

Reply:
I'm indifferent to the extra bits part so whatever you prefer. I think it's good to mention the extra bits for other information but perhaps not saying `nonces made of 192 random bits` because the whole parameter is also called the nonce.

Reply:
Maybe just `With 192 random bits, 64 additional bits...` ?

Reply:
Yep that works for me.

Reply:
Updated, thanks!